flixel/addons/editors/pex/FlxPexParser.hx

```diff
@@ -54,7 +55,7 @@ class FlxPexParser
 		var angle = minMax("angle", config);
 		
 		var startSize = minMax("startParticleSize", config);
-		var finishSize = minMax("finishParticleSize", "FinishParticleSizeVariance", config);
+		var finishSize = minMax("finishParticleSize", "finishParticleSizeVariance", config);
```

Comment:
Oh, I didn't realize there are multiple tools that produce this xml format. Could you add a link to that tool in the class docs, where we link to http://onebyonedesign.com/flash/particleeditor/?